src/views/shared/portal/portal.view.tsx

```diff
@@ -1,19 +1,22 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import usePortalStyles from "./portal.styles";
 
-export const PortalSelector = {
-  FULLSCREEN_MODAL: "#fullscreen-modal-root",
-  SIDENAV: "#sidenav-root",
-};
+import usePortalStyles from "src/views/shared/portal/portal.styles";
 
-function Portal({ selector = PortalSelector.FULLSCREEN_MODAL, children }) {
+export type PortalSelector = "#fullscreen-modal-root" | "#sidenav-root";
```

Comment:
The idea here should be to abstract the selector from the parent components. It's kinda ugly to see this from a parent component: `<Portal selector="#fullscreen-modal-root" />`. Maybe what we can do here is to rename the `selector` property to `target = "full-screen" | "sidenav"`, and then based on this target prop pick one selector or another to use internally in this component.

Reply:
Yes, much better. Done.